website/content/en/status/report-2023-04-2023-06/mfsbsd-building-integration.adoc

```diff
@@ -0,0 +1,30 @@
+=== Integrate mfsBSD into the release building tools
+
+Links: +
+link:https://wiki.freebsd.org/SummerOfCode2023Projects/IntegrateMfsBSDIntoTheReleaseBuildingTools[Wiki Article] URL: link:https://wiki.freebsd.org/SummerOfCode2023Projects/IntegrateMfsBSDIntoTheReleaseBuildingTools[]
+
+Contact: Soobin Rho <soobinrho@FreeBSD.org>
+
+==== Purpose of this project
+
+This project creates an additional target of the weekly snapshots of -current and -stable versions of mfsBSD images in the src/release makefile.
+Currently, only the release versions of mfsBSD images are produced.
+This means they tend to get out of sync with the tools in base.
+This project aims to address that problem.
+
+==== What is mfsBSD?
+
+"mfsBSD is a toolset to create small-sized but full-featured mfsroot based distributions of FreeBSD that store all files in memory (MFS) [Memory File System] and load from hard drive, usb storage device or optical media.
+It can be used for a variety of purposes, including diskless systems, recovery partitions and remotely overwriting other operating systems."
+(link:https://people.freebsd.org/~mm/mfsbsd/mfsbsd.pdf[])
+
+Martin Matuska <mm@FreeBSD.org> is both the author of the white paper above and the maintainer of the mfsBSD repository: link:https://github.com/mmatuska/mfsbsd[]
```

Comment:
Please use the `mailto` macro here:

`mailto:mm@FreeBSD[Martin Matuska]`

And use the link macro as following:

`link:https://github.com/mmatuska/mfsbsd[mfsBSD repository]`.